libraries/core_libs/network/src/tarcap/stats/node_stats.cpp

```diff
@@ -259,8 +261,8 @@ Json::Value NodeStats::getStatus() const {
     }
   }
 
-  if (pbft_syncing_state_->isPbftSyncing()) {
-    res["syncing_from_node_id"] = pbft_syncing_state_->syncingPeer().toString();
+  if (pbft_syncing_state_->isPbftSyncing() && pbft_syncing_state_->syncingPeer()) {
```

Comment:
`pbft_syncing_state_->syncingPeer()` used twice, so maybe better to move it to variable above

Reply:
fixed